docs/release-notes/.FSharp.Compiler.Service/9.0.200.md

```diff
@@ -22,5 +22,6 @@
 * Better ranges for CE `use` error reporting. ([PR #17811](https://github.com/dotnet/fsharp/pull/17811))
 * Better ranges for `inherit` error reporting. ([PR #17879](https://github.com/dotnet/fsharp/pull/17879))
 * Better ranges for `inherit` `struct` error reporting. ([PR #17886](https://github.com/dotnet/fsharp/pull/17886))
+* Better ranges for implicit `inherit` error reporting. ([PR #17893](https://github.com/dotnet/fsharp/pull/17893))
```

Comment:
Sorry for picking on the words - what is _implicit_ about this type of inheritance? I think implicitly we only inherit Object, if nothing is specified 🤔

Reply:
It is how is named in the `AST` `SynMemberDefn.ImplicitInherit`. I guess we use `inherit objects` instead. ?

Reply:
Updated.